Sources/core/objects/Movable.h

```diff
@@ -34,7 +34,7 @@ class IMovable : public IOnMapObject
     void processMove();//for move
     virtual bool TryMove(Dir direct);
     bool CheckMoveTime();
-    bool CheckPassable();
+    virtual bool CheckPassable();
     bool MainMove();
```

Comment:
Yes, virtual keyword -> wrong wording from me. I mean it is needed to remove `virtual` keyword here, and remove this function in `Projectile` class (because now it just copy-paste, and overall it is not needed to change it).